### PR TITLE
Bump gemnasium service gem

### PR DESCRIPTION
**Pull request by gravis**
Hi,

We have released a new version of the service gem. The previous version was using a vulnerable version of the dependency rugged. This fixes the issue related to CVE-2014-9390.

Thanks

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://gemnasium-update-gemnasium-to-0-2-4.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://gemnasium-update-gemnasium-to-0-2-4.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: Gemfile.lock [hunk 1]
```diff
@@ -489,7 +489,7 @@ GEM
     ruby-progressbar (1.7.1)
     rubyntlm (0.4.0)
     rubypants (0.2.0)
-    rugged (0.21.2)
+    rugged (0.21.4)
```
Comment (randx) [inline Gemfile.lock]: If new rugged version will cause errors we might want to revert this line change.

Comment (gravis) [inline Gemfile.lock]: Sure. The only difference is the bundled libgit2.

https://github.com/libgit2/rugged/blob/master/CHANGELOG.md#rugged-0214-january-18-2015

libgit2 changelog is a bit messy, but the 0.21.5 seems to be a maintenance version:
https://github.com/libgit2/libgit2/releases
So I guess 0.21.4 was too.

**Pull request merged**